qiskit_experiments/library/calibration/fine_frequency_cal.py

```diff
@@ -0,0 +1,79 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine frequency calibration experiment."""
+
+from typing import List, Optional
+import numpy as np
+
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.framework import ExperimentData
+from qiskit_experiments.calibration_management import (
+    BaseCalibrationExperiment,
+    BackendCalibrations,
+)
+from qiskit_experiments.library.characterization.fine_frequency import FineFrequency
+
+
+class FineFrequencyCal(BaseCalibrationExperiment, FineFrequency):
+    """A calibration version of the fine frequency experiment."""
+
+    def __init__(
+        self,
+        qubit: int,
+        calibrations: BackendCalibrations,
+        backend: Optional[Backend] = None,
+        repetitions: List[int] = None,
+        auto_update: bool = True,
+    ):
+        r"""see class :class:`FineDrag` for details.
+
+        Note that this class implicitly assumes that the target angle of the gate
+        is :math:`\pi` as seen from the default experiment options.
+
+        Args:
+            qubit: The qubit for which to run the fine drag calibration.
+            calibrations: The calibrations instance with the schedules.
+            backend: Optional, the backend to run the experiment on.
+            auto_update: Whether or not to automatically update the calibrations. By
+                default this variable is set to True.
+        """
+        super().__init__(
+            calibrations,
+            qubit,
+            repetitions,
+            schedule_name=None,
+            backend=backend,
+            cal_parameter_name="qubit_lo_freq",
```

Comment:
Might want to make this more robust using e.g. `cals.qubit_freq_name` or something similar. This is linked to the refactor #541.

Reply:
Done here: 62f08a5